### PR TITLE
[GEN-392] Feedbackform

### PR DESCRIPTION
**Pull request by redpepper-nag**

### CODE REVIEW

**Comment by redpepper-nag** [comment]
IMG_0822
IMG_0823
IMG_0824
IMG_0825

File: trk/src/Screens/Home/index.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ function HomeScreen(props) {
           const documentReference = await addTap(tap);
           const newTapId = documentReference.id;
           console.log("New Tap ID is:", newTapId);
-          navigation.navigate('Detail', { climbData: climbData.data(), tapId: newTapId });
+          navigation.navigate('Detail', { climbData: climbData.data(), tapId: newTapId, climbId: climbId[0] });
```
Comment (VGmt) [inline trk/src/Screens/Home/index.js:73]: what's 'climbId[0]' representing? Maybe clarify this in a future PR.

File: trk/src/Navigation/AppNavigator.js [hunk 2]
```diff
@@ -31,6 +32,11 @@ function HomeStack() {
         component={ClimbDetailScreen}
         options={{ title: 'Climb Detail', headerBackTitle: 'Home', headerTitleAlign: 'center' }}
       />
+      <Stack.Screen
```
Comment (VGmt) [inline trk/src/Navigation/AppNavigator.js:35]: I your point about the Screen being implemented in two parts of the navigator. It should just be one.. Something to figure out in a future PR

**Review by VGmt — APPROVED** [review]
Looks good to me. I added comments on some sections of the code, which aren't blockers to it being approved and merged. go go go

**Pull request merged**